docs/use-cases/community-points.md

```diff
@@ -4,6 +4,8 @@ title: Community Points Engine
 sidebar_label: Community Points Engine
 ---
 
+*By the end of this document you should know how to create a scalable points or rewards system for a given community using the OMG Network.*
```

Comment:
is this appropriate? the document is not exactly structured like a how-to guide

Reply:
True, I just wanted to try it and see your response : )